jobs/payment-jobs/tasks/ejv_partner_distribution_task.py

```diff
@@ -147,8 +165,11 @@ def _create_ejv_file_for_partner(cls, batch_type: str):  # pylint:disable=too-ma
                 if credit_distribution_code.stop_ejv:
                     continue
 
-                line_items = cls._find_line_items_by_invoice_and_distribution(distribution_code_id, invoice_id_list)
-
+                line_items = (
```

Comment:
What if you have `distribution_code` that has partial refunds and normal invoices?

Reply:
How are you marking these partial refunds as disbursed?

Reply:
yes it could be, I updated it to:
`line_items = partial_refund_items + cls._find_line_items_by_invoice_and_distribution(
                        distribution_code_id, invoice_id_list)`